roscala/src/main/scala/coop/rchain/rosette/Ob.scala

```diff
@@ -210,50 +202,44 @@ object Ob {
     }
   }
 
-  def getLex(indirect: Boolean,
-             level: Int,
-             offset: Int): State[Ob, Option[Ob]] = ???
-
-  def getAddr(indirect: Boolean,
-              level: Int,
-              offset: Int): State[Ob, Option[Ob]] = ???
-
-  def setLex(indirect: Boolean,
-             level: Int,
-             offset: Int,
-             value: Ob): State[Ob, StoreResult] = State { ob =>
-    val nthParentLens =
-      (0 until level).foldLeft(lens[Ob]: Lens[Ob, Ob])((l, _) => l >> 'parent)
-
-    def inSlotNum(lens: Lens[Ob, Ob]): Option[Lens[Ob, Ob]] =
-      if (!indirect) Some(lens)
-      else {
-        val numberOfSlots = lens.get(ob).numberOfSlots()
-        if (ob.slotNum() >= numberOfSlots) None
+  def getLex(indirect: Boolean, level: Int, offset: Int): State[Ob, Option[Ob]] = ???
+
+  def getAddr(indirect: Boolean, level: Int, offset: Int): State[Ob, Option[Ob]] = ???
+
+  def setLex(indirect: Boolean, level: Int, offset: Int, value: Ob): State[Ob, StoreResult] =
+    State { ob =>
+      val nthParentLens =
+        (0 until level).foldLeft(lens[Ob]: Lens[Ob, Ob])((l, _) => l >> 'parent)
+
+      def inSlotNum(lens: Lens[Ob, Ob]): Option[Lens[Ob, Ob]] =
+        if (!indirect) Some(lens)
         else {
-          val resLens = unsafeCastLens[Actor](lens) >> 'extension
-          Some(unsafeCastLens[Ob](resLens))
```

Comment:
Is it a change from another PR?

Reply:
I agree with @DoSOfRedRiver , this seem suspicious, almost as if leaked from different branch

Reply:
Hmm, things look fine when you just look at the file in Github.

Reply:
[Hiding whitespace changes](https://github.com/rchain/rchain/pull/234/files?w=1) shows that this is an artifact of the differ, not a real code change, maybe.